ConvertToJpg/ConvertToJpg.sln

```diff
@@ -0,0 +1,25 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 17
+VisualStudioVersion = 17.8.34330.188
```

Comment:
We'll have to modify the instructions in the README slightly to tell customers to open the .sln instead of the .csproj to run the sample.

Reply:
The .sln file was removed because it wasn't necessary to run the sample, so this comment can be disregarded.